deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml

```diff
@@ -46,6 +46,29 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: csi-resizer
+          image: MUSTPATCHWITHKUSTOMIZE
+          imagePullPolicy: Always
+          securityContext:
+            privileged: false
+            allowPrivilegeEscalation: false
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--timeout=600s"
+          env:
+            - name: ADDRESS
+              value: /csi/csi-vpc-file.sock
+          resources:
+            limits:
+              cpu: 100m
```

Comment:
Did we confirm with block resizer and we are using optimal resources

Reply:
Added same resources as block

Reply:
Test